2024-05-21 16:37:08,646 - __main__ - INFO - Lan Audacity - version 1.0.4
2024-05-21 16:38:10,097 - __main__ - INFO - Lan Audacity - version 1.0.4
2024-05-21 16:38:10,680 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'newProject': object of type 'NoneType' has no len()
2024-05-21 16:39:36,343 - __main__ - INFO - Lan Audacity - version 1.0.4
2024-05-21 16:39:36,449 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x000001EF9E4A31D0>
2024-05-21 16:39:36,953 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'newProject': object of type 'NoneType' has no len()
2024-05-21 16:49:24,126 - __main__ - INFO - Lan Audacity - version 1.0.4
2024-05-21 16:49:24,408 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x000001EAD3E13260>
2024-05-21 16:49:25,395 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'newProject': object of type 'NoneType' has no len()
2024-05-21 16:49:25,396 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'openProject': object of type 'NoneType' has no len()
2024-05-21 16:49:25,396 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'saveProject': object of type 'NoneType' has no len()
2024-05-21 16:49:25,396 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'saveAsProject': object of type 'NoneType' has no len()
2024-05-21 16:49:25,397 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'closeProject': object of type 'NoneType' has no len()
2024-05-21 16:49:25,397 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'exit': object of type 'NoneType' has no len()
2024-05-21 16:49:25,399 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'undoAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,399 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'redoAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,399 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'cutAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,400 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'copyAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,400 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'pasteAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,400 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'deleteAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,401 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'selectAllAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,401 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'selectNoneAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,402 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'selectInverseAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,402 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'fileExplorerAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,402 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x000001EAD6CD0680>
2024-05-21 16:49:25,403 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'extensionAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,403 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'userAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,403 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'preferencesAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,404 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'languageAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,404 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'shortcutKeyAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,404 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x000001EAD6CD0C20>
2024-05-21 16:49:25,405 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'terminalAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,405 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'helpAction': object of type 'NoneType' has no len()
2024-05-21 16:49:25,405 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'aboutAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,569 - __main__ - INFO - Lan Audacity - version 1.0.4
2024-05-21 20:48:21,755 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x102b487a0>
2024-05-21 20:48:21,882 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'newProject': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'openProject': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'saveProject': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'saveAsProject': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'closeProject': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'exit': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'undoAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,883 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'redoAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'cutAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'copyAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'pasteAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'deleteAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'selectAllAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'selectNoneAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'selectInverseAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'fileExplorerAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,884 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x102b49e20>
2024-05-21 20:48:21,889 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'extensionAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,889 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'userAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,889 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'preferencesAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,889 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'languageAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,890 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'shortcutKeyAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,890 - root - DEBUG - <PyQt5.QtGui.QIcon object at 0x102b4a3c0>
2024-05-21 20:48:21,890 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'terminalAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,890 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'helpAction': object of type 'NoneType' has no len()
2024-05-21 20:48:21,890 - __main__ - ERROR - Une erreur inattendue s'est produite lors de la récupération de l'icône 'aboutAction': object of type 'NoneType' has no len()
2024-05-21 22:22:05,024 - __main__ - INFO - Lan Audacity - version 1.0.4
2024-05-21 22:22:22,613 - __main__ - INFO - Lan Audacity - version 1.0.4
